ansible_collections/arista/avd/requirements-minimum.txt

```diff
@@ -0,0 +1,15 @@
+# requirements-minimum.txt is leveraged in the CI pipeline to test and qualify all our minimum requirements.
```

Comment:
would probably be good to be able to build this programmatically from out requirement file